tests/data/short_circuit/single_phase_to_ground/params.json

```diff
@@ -0,0 +1,5 @@
+{
+  "calculation_method": "short_circuit",
```

Comment:
here and below: this is incorrect and outdated. this should be `ie60909`.

did you make sure to run `python -m pip install .` on the `pgm-test-generator` and/or merge with main before running `python main.py` on the `pgm-test-generator`?

maybe it's a good idea to just clean your build environment and/or `venv` on the `pgm-test-generator` and start with a clean workspace

Reply:
Yeah, there was a merge conflict ;(